src/components/QuranReader/TafsirView/TafsirText/index.tsx

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable react/no-danger */
+import React from 'react';
+
+import { useDispatch, useSelector } from 'react-redux';
+
+import styles from './TafsirText.module.scss';
+
+import Counter from 'src/components/dls/Counter/Counter';
+import {
+  MAXIMUM_FONT_STEP,
+  MINIMUM_FONT_STEP,
+  selectQuranReaderStyles,
+  increaseTafsirFontScale,
+  decreaseTafsirFontScale,
+} from 'src/redux/slices/QuranReader/styles';
+import { logValueChange } from 'src/utils/eventLogger';
+
+type Props = {
```

Comment:
nit: `TafsirTextProps`?
I actually prefer `Props` lol, but other components seems to have component name prefix

Reply:
Well the components I create always have `Props` 😆 but we should standardize this. Will change to `TafsirTextProps` for now